pkg/3scale/amp/template/adapters/system.go

```diff
@@ -1,6 +1,8 @@
 package adapters
 
 import (
+	"fmt"
```

Comment:
For 2.11 we need to remember that we'll need to write a templates upgrade procedure for the changes related to this PR